corehq/apps/cloudcare/views.py

```diff
@@ -303,8 +302,9 @@ class PreviewAppView(TemplateView):
     @use_daterangepicker
     @xframe_options_sameorigin
     def get(self, request, *args, **kwargs):
-        if should_restrict_web_apps_usage(request.domain):
-            context = get_context_for_ucr_limit_error(request.domain)
+        mobile_ucr_count = get_mobile_ucr_count(request.domain)
+        if should_restrict_web_apps_usage(request.domain, mobile_ucr_count):
+            context = BlockWebAppsView.get_context_for_ucr_limit_error(request.domain, mobile_ucr_count)
             return render(request, 'preview_app/block_app_preview.html', context)
```

Comment:
@gherceg per our conversation earlier, you can actually just use `set_bootstrap_version5()` here rather than making a new view

Reply:
@biyeun I played around with this but was unsuccessful in getting it to work. I'm tempted to just continue using the redirect option currently implemented in this PR.